batch_report_pdf_zip/models/ir_actions_report.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2024 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class IrActionsReport(models.Model):
+    _inherit = "ir.actions.report"
+
+    is_zip = fields.Boolean(
```

Comment:
To be a bit more descriptive.
```suggestion
    zip_download = fields.Boolean(
```